README.md

```diff
@@ -18,5 +19,68 @@ a basic chatting application.
 <img src="./images/structure.png" alt="structure"/>
 
 ## Initial DB Schema
+
 <img src="./images/initial-schema.png" alt="db Schema"/>
 
+## Prerequisites
+
+Before setting up the Community App, ensure that you have the following prerequisites installed:
+
+- Node.js (version 14 or above)
+- MongoDB
+
+## Installation
+
+Follow the steps below to set up the Community App:
+
+**1. Clone the repository:**
+
+```bash
+git clone https://github.com/Codepire/community-app-backend.git
+```
+
+**2. Navigate to the project directory:**
+
+```bash
+cd community-app-backend
+```
+
+**3. Install with:**
+
+```bash
+npm install
+```
+
+**4. Configure environment variables:**
+
+- Create a .env file in the root directory.
+- Add the following variables to the .env file:
+
+```bash
```

Comment:
update env variables with these.
for example
  
JWT_SEED=aslkjfa;sldfjk
TOKEN_EXPIRATION_TIME=1h